test/test_dispatch.py

```diff
@@ -935,5 +935,20 @@ def test_defaultbackend_math(self):
                 r"Registration to both CompositeImplicitAutograd and CompositeExplicitAutograd is not allowed"):
             dispatcher.register(["CompositeExplicitAutograd", "CompositeImplicitAutograd"])
 
+    def test_quantized_structured_not_implemented(self):
```

Comment:
can we add a test in https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_tensor.py#L142 for calling some methods (e.g. qscheme) on an unknown tensor as well?

Reply:
sure